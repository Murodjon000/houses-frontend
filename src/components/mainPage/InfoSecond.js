import React from 'react';
import { Link } from '@reach/router';
import houses from '../../assets/houses.png';

const Info = () => (
  <div className=" info__wrapper pt-5 px-4 bg-blue-prime">
    <div className="row align-items-center">
      <div className=" col-md-6 info__wrapper-image">
        <img src={houses} alt="Houes" />
      </div>
      <div className="col-md-6 info__wrapper-info ">
        <h1 className="text-lg-black">Only best places for your rest</h1>
        <p className="text-sm-gray mb-5 mt-3">
          More than ten thousand houses at your disposal.Only the best and
          checked landlords.We will find best place for you and the conditions
          for a safe transaction and comfortable stay.
        </p>
        <Link to="/signup" className="main-btn mt-4">
          START FREE
        </Link>
      </div>
    </div>
  </div>
);

export default Info;
