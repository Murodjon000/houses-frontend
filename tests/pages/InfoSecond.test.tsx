import React from 'react';
import renderer from 'react-test-renderer';
import InfoSecond from '../../src/components/mainPage/InfoSecond';

test('should correctly render Details page', () => {
  const component = renderer.create(<InfoSecond image="https://images.unsplash.com/photo-1603993097397-89c963e325c7?ixid=MnwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
