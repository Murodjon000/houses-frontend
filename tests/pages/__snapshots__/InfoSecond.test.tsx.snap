// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Details page 1`] = `
<div
  className=" info__wrapper pt-5 px-4 bg-blue-prime"
>
  <div
    className="row align-items-center"
  >
    <div
      className=" col-md-6 info__wrapper-image"
    >
      <img
        alt="Houes"
        src="https://images.unsplash.com/photo-1603993097397-89c963e325c7?ixid=MnwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80"
      />
    </div>
    <div
      className="col-md-6 info__wrapper-info "
    >
      <h1
        className="text-lg-black"
      >
        Only best places for your rest
      </h1>
      <p
        className="text-sm-gray mb-5 mt-3"
      >
        More than ten thousand houses at your disposal.Only the best and checked landlords.We will find best place for you and the conditions for a safe transaction and comfortable stay.
      </p>
      <a
        className="main-btn mt-4"
        href="/signup"
        onClick={[Function]}
      >
        START FREE
      </a>
    </div>
  </div>
</div>
`;
